frontend/src/metabase/ui/components/feedback/Skeleton/Skeleton.tsx

```diff
@@ -1,3 +1,19 @@
-export type { SkeletonProps } from "@mantine/core";
-export { Skeleton } from "@mantine/core";
-export { getSkeletonOverrides } from "./Skeleton.styled";
+import type { SkeletonProps } from "@mantine/core";
+import { Skeleton as MantineSkeleton } from "@mantine/core";
+import { useMemo } from "react";
+
+export const Skeleton = ({
+  natural,
+  ...props
+}: SkeletonProps & {
+  /** Automatically assign a natural-looking, random width to the skeleton */
+  natural?: boolean;
+}) => {
+  const width = useMemo(
+    () => (natural ? `${Math.random() * 30 + 50}%` : props.width),
```

Comment:
```suggestion
    () => {
      if (!natural) return props.width;
      // validate naturalMin < max somewhere above 
      const spread = naturalMax - naturalMin;
      return `${_.random(0, spread) + naturalMin}%`
    }
```
Would be cool if the min/max were configurable. Also using lodash to get round numbers might nice.